zipline/finance/performance/tracker.py

```diff
@@ -296,6 +296,8 @@ def process_trade(self, event):
                 perf_period.handle_cash_payment(cash_adjustment)
 
     def process_transaction(self, event):
+        # Transaction events also count as trade events for the trackers
+        self.process_trade(event)
```

Comment:
Though the logic applied is the same, calling `process_trade` here is confusing.

Not sure what the right fix for that would be, some candidates. 1) would be 'inlining' via copy/paste the handle cashe logic into process_transaction. 2) Or make a function with some common name that both process_trade and process_transaction call.